x/group/keeper/msg_server.go

```diff
@@ -185,6 +185,11 @@ func (k Keeper) UpdateGroupMembers(goCtx context.Context, req *group.MsgUpdateGr
 		// Update group in the groupTable.
 		g.TotalWeight = totalWeight.String()
 		g.Version++
+
+		if err := k.validateDecisionPolicies(ctx, *g); err != nil {
```

Comment:
I'm adding `DecisionPolicy.Validate()` calls in a few places in the msg server:
- MsgCreateGroupPolicy and MsgCreateGroupWithPolicy
- MsgUpdateDecisionPolicy
- MsgLeaveGroup
- MsgUpdateGroupMembers

anywhere else you think we need to add validation?